pkg/webhooks/resource/validation/validation.go

```diff
@@ -178,8 +178,10 @@ func (v *validationHandler) handleAudit(
 		gv := metav1.GroupVersion{Group: request.Kind.Group, Version: request.Kind.Version}
 		controllerutils.SetOwner(report, gv.String(), request.Kind.Kind, resource.GetName(), resource.GetUID())
 	}
-	_, err = reportutils.CreateReport(context.Background(), report, v.kyvernoClient)
-	if err != nil {
-		v.log.Error(err, "failed to create report")
+	if len(report.GetResults()) > 0 {
```

Comment:
Do we need this in the background scan?

Reply:
Not necessarily, there's a 1 to 1 mapping between the resource and the report with background scans.